src/query/expression/src/values.rs

```diff
@@ -370,7 +370,12 @@ impl Scalar {
                 let col = builder.build();
                 Scalar::Map(col)
             }
-            DataType::Bitmap => Scalar::Bitmap(vec![]),
+            DataType::Bitmap => {
+                let mut bytes = vec![];
+                let rb = RoaringTreemap::new();
+                rb.serialize_into(&mut bytes).unwrap();
+                Scalar::Bitmap(bytes)
```

Comment:
put empty bytes as the default value is ok, we should check whether it is empty when deserializing the binary bitmap.

Reply:
fine